src/dbally/views/pandas_base.py

```diff
@@ -33,20 +32,24 @@ async def apply_filters(self, filters: IQLQuery) -> None:
         """
         Applies the chosen filters to the view.
 
-        :param filters: IQLQuery object representing the filters to apply
+        Args:
+            filters: IQLQuery object representing the filters to apply
         """
         self._filter_mask = await self.build_filter_node(filters.root)
 
     async def build_filter_node(self, node: syntax.Node) -> pd.Series:
         """
-        Converts a filter node from the IQLQuery to a Pandas Series representing
-        boolean mask to be applied to the dataframe.
+        Converts a filter node from the IQLQuery to a Pandas Series being
```

Comment:
Not that important but I preferred "representing" - the object is not inherently a mask, it is just a boolen DataFrame, that _represents_ a mask.